pkg/daemon/daemon.go

```diff
@@ -544,8 +574,14 @@ func (dn *Daemon) restartDevicePluginPod() error {
 }
 
 func (dn *Daemon) loadVendorPlugins(ns *sriovnetworkv1.SriovNetworkNodeState) error {
-	pl := registerPlugins(ns)
-	pl = append(pl, GenericPlugin)
+	var pl []string
```

Comment:
I think we can re-use the same registerPlugins functions for both virtual and baremetal by moving this piece of code into registerPlugins and pass an additional argument to it. for example:

```
registerPlugins(ns, dn.platform)
```

Reply:
The problem is that with virtual, we don't want to load the generic plugin.

Reply:
Yes, you're right, I forgot the generic-plugin is not needed in virtual env.